src/in-cell-bar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+
+import { makeStyles } from '@eqworks/lumen-labs'
+
+
+const useStyles = ({ bgColor, barLength }) => makeStyles({
+  bar: {
+    backgroundColor: bgColor,
+    height: '14px',
+    width: `${barLength}%`,
+    marginRight: '4px',
+  },
+  value: {
+    fontWeight: 400,
+    fontSize: '12px',
+    lineHeight: '14px',
+    textAlign: 'center',
+    color: '#6C6C6C', // secondary-700
+  },
+})
+
+const maxValsPerColumn = {}
+const computeMaxVals = (data, columnID, maxValsPerColumn) => {
+  if (!maxValsPerColumn[columnID]) {
+    const max = Math.max(...data.map((column) => parseInt(column[columnID])))
+    maxValsPerColumn[columnID] = max
+  }
+  return maxValsPerColumn
+}
+const computeBgIntensity = (value, range) => {
```

Comment:
@kc-leung it looks like Selin uses 4 shades of colour only, so this would be where I would use the d3 lineare scale function.

Reply:
yes, and it should be dynamic